### PR TITLE
[Ready for merge] Updating "Data Stewards" page with figure

### LINKED ISSUE #3375: [Image needs adding] Data Stewards Chapter
### Summary

The chapter: https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles/data-steward.html does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

datastewards

### What needs to be done?

- [ ] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [ ] Write the alt text for the image
- [ ] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### PR DESCRIPTION
**Pull request by kallewesterling**
Adding figure and alt-text to the "Data Stewards" page. Fixes #3375.

### CODE REVIEW

**Comment by kallewesterling** [comment]
For reference, #3331.

File: book/website/collaboration/research-infrastructure-roles/data-steward.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 Data Stewards are a growing role within scientific communities.
 'Data Steward' is an umbrella term for numerous support roles that involve the creation, management and usage of research data (see also the {ref}`rr-rdm` chapter).
 
+```{figure} ../../figures/data-stewards.*
+---
+height: 400px
+name: data-stewards
+alt: Cartoon-like sketch of two characters depicted as data stewards in superhero attire, with the left figure gesturing towards symbols of knowledge and alert, and the right figure pointing to tools of the trade such as security and connectivity, all encompassed by a theme of data management and protection.
```
Comment (Arielle-Bennett) [inline book/website/collaboration/research-infrastructure-roles/data-steward.md:11]: I would add in a description of the colour scheme

Comment (Arielle-Bennett) [inline book/website/collaboration/research-infrastructure-roles/data-steward.md:11]:
```suggestion
alt: Black, white, grey and purple, cartoon-like sketch of two characters depicted as data stewards in superhero attire, with the left figure gesturing towards symbols of knowledge and alert, and the right figure pointing to tools of the trade such as security and connectivity, all encompassed by a theme of data management and protection.
```

Comment (kallewesterling) [reply book/website/collaboration/research-infrastructure-roles/data-steward.md]: Thank you @Arielle-Bennett!

**Review by Arielle-Bennett — APPROVED** [review]
Approved!

**Comment by kallewesterling** [comment]
Updating title to reflect it - thanks @Arielle-Bennett !

**Comment by kallewesterling** [comment]
@all-contributors add kallewesterling content,review

**Pull request merged**